docs/modules/operation/pages/provisioning/snmp-profile.adoc

```diff
@@ -36,14 +34,14 @@ In the above config,
 1. `profile1` doesn't have a filter expression.
 This profile will be tried for every interface.
 
-2. `profile2` has a filter expression that compares `iphostname` (the hostname resulting from a reverse DNS lookup of the IP address being fitted) against a preconfigured value.
+2. `profile2` has a filter expression that compares `iphostname` (the hostname resulting from a reverse DNS lookup of the IP address being fitted) against a pre-configured value.
```

Comment:
"preconfigured" is actually standard use according to Oxford and Webster's.

Reply:
```suggestion
2. `profile2` has a filter expression that compares `iphostname` (the hostname resulting from a reverse DNS lookup of the IP address being fitted) against a preconfigured value.
```